recipes/public-wrappers/meta.yaml

```diff
@@ -0,0 +1,56 @@
+{% set name = "public-wrappers" %}
+{% set version = "0.2.1" %}
+{% set sha256 = "f2173dc480ea8b2baf59b2811d19783ea86005aabf2a7fdbb284a2a752d04a6a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://github.com/tesujimath/{{ name }}/archive/v{{ version }}.tar.gz
```

Comment:
From the error or CircleCI, it doesn't look like you can build this from the github archive tarball. We prefer that you don't pull source from a git repo:

https://conda-forge.org/docs/meta.html#build-from-tarballs-not-repos

I'm not sure what the best solution is here. Is it feasible to submit the project to pypi?

Reply:
Yep, project is on PyPI.  I'll try that as the origin for the tarball.